### ISSUE TITLE
Create customer portal session with flow subscription_update - property 'subscription_update' does not exist in type 'FlowData'

### ISSUE THREAD

**Issue by chladnefazole** [stripe/stripe-node#2197]
### Describe the bug

TS complains about missing property 'subscription_update' on interface FlowData

### To Reproduce

I want customers to be able to upgrade their subscriptions, so I followed the docs to create a portal session: https://docs.stripe.com/api/customer_portal/sessions/create

I try to call the function like this:
````
                Stripe.billingPortal.sessions.create({
                    customer: req.session.user.customerId,
                    return_url: `${domain}account/${req.session.user.id}`,
                    flow_data: {
                        type: 'subscription_update' as stripe.BillingPortal.SessionCreateParams.FlowData.Type,
                        subscription_update: {
                            subscription: stripeSubscriptionId,
                        }
                    }
                }).then((stripeSession) => {
                    res.redirect(303, stripeSession.url);
                });
````

But I get a TS error:
![image](https://github.com/user-attachments/assets/dafd304c-8bf9-4612-9f64-6257a4b9b3dc)

And in the FlowData interface I can see there is no property 'subscription_update':
![image](https://github.com/user-attachments/assets/20446e58-5ddf-4175-a572-a9d17291c3e7)

I can get around the error by declaring the flow data object separately, but it's messier than passing the object in directly:
````
                const flow_data = {
                    type: 'subscription_update' as stripe.BillingPortal.SessionCreateParams.FlowData.Type,
                    subscription_update: {
                        subscription: stripeSubscriptionId,
                    }
                };
                Stripe.billingPortal.sessions.create({
                    customer: req.session.user.customerId,
                    return_url: `${domain}account/${req.session.user.id}`,
                    flow_data
                }).then((stripeSession) => {
                    res.redirect(303, stripeSession.url);
                });
````

### Expected behavior

Property 'subscription_update' should exist on interface FlowData.

### OS

Windows 11

### Node version

v20.11.0

### Library version

1.21.6

### API version

2024-09-30.acacia

**Comment by remi-stripe** [comment]
@chladnefazole I looked at the code and I see it defined [here](https://github.com/stripe/sdk-codegen/blob/5b2d9d7c/generated/node-next/types/BillingPortal/SessionsResource.d.ts.generated#L58) as expected. 
You said you are on the SDK version 1.21.6 which doesn't make sense as it was years before we added Typescript support. But my guess is that you are either using external types or you are on a version of stripe-node that is too old and predates the addition of that feature in the API.
